core/identity-hub-core/src/main/java/org/eclipse/edc/identityhub/DefaultServicesExtension.java

```diff
@@ -77,6 +85,12 @@ public void initialize(ServiceExtensionContext context) {
 
         var scopeIsPresentRule = new ClaimIsPresentRule(ACCESS_TOKEN_SCOPE_CLAIM);
         registry.addRule(DCP_ACCESS_TOKEN_CONTEXT, scopeIsPresentRule);
+
+        if (context.getSetting(ACCESSTOKEN_JTI_VALIDATION_ACTIVATE, false)) {
```

Comment:
`false` could be extracted as a constant and used also in the `@Setting` annotation